src/Pos/Wallet/Web/BListener.hs

```diff
@@ -66,8 +70,21 @@ onApplyTracking blunds = do
                        trackingApplyTxs encSK allAddresses $
                        zip txs (repeat newTip)
         applyModifierToWallet wAddr newTip mapModifier
+        addrs <- either addrDecodeErr pure $
```

Comment:
it's implied you to add logic to `trackingApplyTxs` and add a new field into `CAccModifier` and can use `trackingApplyTxs` everywhere, for example, in order to consider mempool